src/lib/apps/tokens/Tokens.ts

```diff
@@ -1,19 +1,24 @@
 import * as Types from './types';
 import { SubResource } from '../../base';
 import { APIEndpoint } from '../../APIEndpoint';
+import { TokenRequest } from '../types';
 
 class AppsTokens extends SubResource {
-  refresh = APIEndpoint<Types.RefreshTokenPayload, Types.RefreshTokenResponse>({
+  create = APIEndpoint<TokenRequest<Types.RefreshTokenPayload>, Types.RefreshTokenResponse>({
```

Comment:
You actually don't need a token to refresh the authorization code, since this call doesn't affect client sites.